package.json

```diff
@@ -33,6 +33,7 @@
     "date-fns": "^1.30.1",
     "material-icons": "^0.2.3",
     "moment": "^2.24.0",
+    "netlify-identity-widget": "github:iteatime/netlify-identity-widget#staging",
```

Comment:
Staging is just for testing, you will have some normal tag after ?

Reply:
Yes